<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>C059 [urw]</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MEETINGS PAGE</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059 [UKWN]</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MEETINGS AND EVENTS HISTORY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RECORD NEW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059 [urw]</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FILTER BY:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ALL DATES</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2025-02-11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2024-12-11</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>80</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ALL AGENDAS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MONTHLY MEETING</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EMERGENCY MEETING</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>80</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>APPLY FILTER</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>1361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>1141</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059 [urw]</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>163</number>
   </attribute>
   <column>
    <property name="text">
     <string>Meeting Label</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Meeting Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Meeting Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Agenda</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total Contribs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Facilitator</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Attendance</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="view_meeting">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>200</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>View Meeting Details</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>250</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
    </font>
   </property>
   <property name="text">
    <string>(select a meeting then click)</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
